package.json

```diff
@@ -41,6 +41,7 @@
     "@types/qs": "^6.9.7",
     "ajv": "^6.12.6",
     "async": "^3.2.0",
+    "axios": "^1.2.2",
```

Comment:
I'd use https://www.npmjs.com/package/node-fetch or similar so when Node.js introduces stable support for `fetch` the code will not be different.